app/controllers/allFilter.js

```diff
@@ -0,0 +1,17 @@
+const User = require("../models/User.js");
```

Comment:
I should call this file something like `filterUsers.js`, see our [Contributing.md](https://github.com/kriskuiper/project-tech-team/blob/development/docs/CONTRIBUTING.md#make-functions-say-what-theyre-intended-to-do)